interfaces/nav_board.py

```diff
@@ -33,13 +33,12 @@ def __init__(self):
 
     def process_imu_data(self, packet):
         self._pitch, self._heading, self._roll = packet.data
+        self._heading = self._heading
```

Comment:
Why bother to do self._heading = self._heading

Reply:
Probably artifact from some past testing. Will fix